packages/components/src/services/ForumService.ts

```diff
@@ -0,0 +1,55 @@
+import { HttpService } from '../utils/httpService';
+import { CreateQuestion, ForumPosts } from '../schemas/Forum.schema';
+
+const apiUrl = 'https://breakup-app-api.herokuapp.com/api';
+
+export const errorMessages = {
+  failedToLogin: 'Failed to login',
+} as const;
+
+export class ForumService {
+  constructor(private httpService: HttpService) {}
+
+  async fetchPosts(currentPage: number): Promise<ForumPosts[]> {
+    const url = `${apiUrl}/home/forum/${currentPage}`;
+    try {
+      const {
+        data: { questions },
+      } = await this.httpService.get<any>(url);
```

Comment:
in `await this.httpService.get<any>(url); `replace `any` with the schema

Reply:
added todo for further thinking , jab page implement hoga